pkgs/tools/system/tiptop-py/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, fetchFromGitHub
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "tiptop-py";
+  version = "0.2.8";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "nschloe";
+    repo = "tiptop";
+    rev = "v${version}";
+    hash = "sha256-3nrYwcdqGGRvYB5FOW/OCqlRHIlCdkxClBfyKnVC1rQ=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    setuptools
+    py-cpuinfo
+    distro
+    psutil
+    rich
+    textual
+  ];
+
+  checkInputs = [ python3.pkgs.pytest ];
+
+  checkPhase = ''
+    pytest
+  '';
+
+  pythonImportsCheck = [
+    "tiptop"
+  ];
+
+  postInstall = ''
+    mv $out/bin/tiptop $out/bin/tiptop-py
```

Comment:
Any reason for this?

Reply:
Maybe it looks more consistent.

Reply:
Consistent with what?

Reply:
```
 tiptop = callPackage ../os-specific/linux/tiptop { };
 tiptop-py = callPackage ../tools/system/tiptop-py { };
```
Could you give me some advice name collision?

Reply:
Hum, understood.

_meta.priority = <some int>;_ can be used to circumvent this. Smaller numbers indicate highest priority, so that the conflict will be solved by "shadowing" the lowest priority.

Even better, in `all-packages.nix` you can put `tiptop-py = hiPrio (callPackage ...);`, this function will set a higher-than-default priority.

Reply:
Thank you, looks good right now.